frontend/src/hooks.shared.ts

```diff
@@ -20,7 +20,12 @@ export function getErrorMessage(error: unknown): string {
   );
 }
 
-export function validateFetchResponse(response: Response, isAtLogin: boolean, isHome: boolean): void {
+export function validateFetchResponse(
+  response: Response, isAtLogin: boolean, isHome: boolean, config?: LexboxResponseHandlingConfig): void {
```

Comment:
we could probably remove `isAtLogin` and use the new configuration property for the login request in user.ts

Reply:
Made the change